routes/api/index.js

```diff
@@ -6,6 +6,7 @@ router.use('/assets', require('./assets'));
 router.use('/auth', require('./auth'));
 router.use('/passwords', require('./passwords'));
 router.use('/resources', require('./resources'));
+router.use('/questionnaire', require('./questionnaire'));
 router.use('/users', require('./users'));
 
```

Comment:
@JesusEArambula in this file, you need to link up the form path (i.e. `/forms`) to the file...

Reply:
This is the issue that I think is causing the 404 on the forms api...